mobify/platform-scaffold
# Track latest patch of SDK without a release

This PR changes the `package.json` dependency on the SDK so that *this* project will get the latest patch version of the SDK when `npm install` is run.  This means that we no longer have to do a release of the SDK **unless** the scaffold is using a feature introduced in a new version of the SDK. 

 **JIRA**: N/A
 **Linked PRs**: N/A

## Changes
- Change package.json version spec to use a [`~`](https://docs.npmjs.com/misc/semver#tilde-ranges-123-12-1) prefix.

## How to test-drive this PR
- Update the SDK version in package.json manually to `0.14.5`
- Run `rm -rf node_modules && npm install && cat node_modules/progressive-web-sdk/package.json | grep 'version'`
- **Verify**: Ensure that the version is `0.14.5`
- Run `git reset --hard head`
- Run `rm -rf node_modules && npm install && cat node_modules/progressive-web-sdk/package.json | grep 'version'`
- **Verify**: Ensure that the version is `0.14.7`
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
